selftests/unit/test_loader.py

```diff
@@ -516,7 +516,9 @@ def test_double_import(self):
         path = os.path.join(os.path.dirname(os.path.dirname(__file__)),
                             '.data', 'loader_instrumented', 'double_import.py')
         tests = self.loader.discover(path)
-        exps = [('Test2', 'selftests/.data/loader_instrumented/double_import.py:Test2.test2'),
+        exps = [('Test1', 'selftests/.data/loader_instrumented/double_import.py:Test1.test1'),
```

Comment:
Seems now it's not broken anymore, right? Doesn't make sense remove the comment at the beginning of this test?

Reply:
Right! Fixing that!